qa/workunits/ceph-helpers.sh

```diff
@@ -513,6 +524,9 @@ function destroy_osd() {
     local dir=$1
     local id=$2
 
+    # First mask osd down, otherwise it cannot be deleted later on
+    # is status has not propagated to monitor.
```

Comment:
I only added the ` [ x`uname`x = xFreeBSDx ] || send_signal=0`.
The part about the OSD maks is not mine. Perhaps a left over from rebasing earlier on.
I will remove that piece of comments.

But this is the wrong place... __teardown_btrfs will not be called on a FreeBSD platform.